extension/docs/HowToRun.md

```diff
@@ -43,113 +43,60 @@ Extension module requires 1 environment variable to start. This environment vari
-| `keepAliveTimeout`           | Milliseconds to keep a socket alive after the last response ([Node.js docs](https://nodejs.org/dist/latest/docs/api/http.html#http_server_keepalivetimeout)).                                                                                                                                                                                                                                                            | NO       | Node.js default (5 seconds)                                                                                                  |
-| `basicAuth`                  | Boolean attribute to enable/disable basic authentication to prevent unauthorized 3rd-party from accessing extension endpoint                                                                                                                                                                                                                                                                                             | NO       | false                                                                                                                        |
-| `removeSensitiveData`        | Boolean attribute. When set to "false", Adyen fields with additional information about the payment will be saved in the interface interaction and in the custom fields.                                                                                                                                                                                                                                                  | NO       | true                                                                                                                         |
-| `addCommercetoolsLineItems`  | Boolean attribute. If set to **true**, integration will add lineItems to payment methods that require lineItems on `makePaymentRequest`. Payment method types that requires [lineItems](https://docs.adyen.com/api-explorer/#/CheckoutService/latest/payments__reqParam_lineItems): `klarna`, `affirm`, `afterpay`, `afterpaytouch`, `klarna`, `ratepay`, `facilypay`, `clearpay`, `grabpay`, `paybright`, `pix`, `zip`. | NO       | false                                                                                                                        |
+> Note that, multiple child attributes can be provided in the `adyen` and `commercetools` attribute group. Other configurations can be set as direct child attributes in `ADYEN_INTEGRATION_CONFIG`.
```

Comment:
```suggestion
> Note that multiple child attributes can be provided in the `adyen` and `commercetools` attribute group. Other configurations can be set as direct child attributes in `ADYEN_INTEGRATION_CONFIG`.
```

https://forum.english.best/t/comma-or-no-comma-after-note-that/24557, https://www.englishforums.com/English/CommaAfterNoteThat/jlkb/post.htm